tests/backend_test.py

```diff
@@ -1132,21 +1119,21 @@ def test_cloud_stabiliser(simulator_stabilizer_backend: IBMQBackend) -> None:
 
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
-def test_available_devices() -> None:
-    backend_info_list = IBMQBackend.available_devices(
-        hub="ibm-q", group="open", project="main"
-    )
+def test_available_devices(ibm_provider: IBMProvider) -> None:
+    backend_info_list = IBMQBackend.available_devices(instance="ibm-q/open/main")
     assert len(backend_info_list) > 0
 
-    provider = IBMQ.providers(hub="ibm-q", group="open")[0]
+    # Check consistency with pytket-qiskit and qiskit provider
+    assert len(backend_info_list) == len(ibm_provider.backends())
```

Comment:
This test ensures that the length of the list given by `IBMQBackend.available_devices()` is the same as the length of the list of devices in `IBMProvider.backends()` with the same `instance` string used in both. This doesn't check anything about the devices in the list only that the lengths match. 

Is this check thorough enough for testing purposes?